scripts/ci.sh

```diff
@@ -1,38 +0,0 @@
-#!/bin/bash
-set -e
-
-# https://classic.yarnpkg.com/en/docs/cli/policies/#toc-policies-set-version
-# This could be removed if all TC agents guaranteed global yarn installed
-shopt -s expand_aliases
-PROJECT_ROOT=$(pwd)
-alias yarn='node ${PROJECT_ROOT}/.yarn/releases/yarn-*.*js'
```

Comment:
If this has been removed, do we still need the vendored version of yarn in this repo?